src/framerate_type.h

```diff
@@ -0,0 +1,67 @@
+/* $Id$ */
+
+/*
+* This file is part of OpenTTD.
+* OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+* OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+* See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef FRAMERATE_GUI_H
+#define FRAMERATE_GUI_H
+
+#include "stdafx.h"
+#include "core/enum_type.hpp"
+
+enum PerformanceElement {
+	PFE_GAMELOOP,        ///< Speed of gameloop processing.
+	PFE_FIRST = PFE_GAMELOOP,
+	PFE_ACC_GL_ECONOMY,  ///< Time spent processing cargo movement
+	PFE_ACC_GL_TRAINS,   ///< Time spent processing trains
+	PFE_ACC_GL_ROADVEHS, ///< Time spend processing road vehicles
+	PFE_ACC_GL_SHIPS,    ///< Time spent processing ships
+	PFE_ACC_GL_AIRCRAFT, ///< Time spent processing aircraft
+	PFE_GL_LANDSCAPE,    ///< Time spent processing other world features
```

Comment:
The Landscape element actually accumulates across three different functions, but not over a number of similar objects being processed. Should it have the ACC infix? Should the ACC infix even exist?